ROMFS/px4fmu_common/mixers/BigFat.main.mix

```diff
@@ -0,0 +1,43 @@
+Mixer for an YL VTOL
+=======================
+Throttle mixer
+--------------
+M: 1
+O:      10000  10000      0 -10000  10000
+S: 0 3      0  20000 -10000 -10000  10000
+
+Reserved
+---------------
+Z:
+
+Aileron 1 mixer(Left)
+---------------
+M: 1
+O:       10000  10000      	0 -10000  10000
```

Comment:
Did this file have something wrong?   I checked and  revised this file many times. I still don't know why it is not work.